server.js

```diff
@@ -19,7 +19,7 @@ app.engine("handlebars", exphbs({ defaultLayout: "main" }));
 app.set("view engine", "handlebars");
 
 
-app.use(require("./routes/api-routes.js"));
+app.use('/api', require("./routes/api-routes.js"));
```

Comment:
This makes it so all the routes in your api-routes are prefixed with `/api`. So the `/new` route in api-routes is actually accessed with `/api/new`